guides/common/assembly_understanding-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,23 @@
+include::modules/con_understanding-multiple-content-view-environments.adoc[]
```

Comment:
I recommend that you not use this module for the assembly title and intro. Most of the modules below are procedures, not concepts. 

I recommend creating a concept module called "Managing multiple content views" with an overview of the assembly's contents. That title indicates the procedural content of this assembly and is consistent with the title of the preceding assembly, "Managing content views".